pinnwand/http.py

```diff
@@ -198,14 +206,23 @@ class APINew(Base):
             )
             raise tornado.web.HTTPError(400)
 
-        paste = database.Paste(raw, lexer, utility.expiries[expiry])
+        paste = database.Paste(
+            raw, lexer, utility.expiries[expiry], description
+        )
 
         with database.session() as session:
             session.add(paste)
             session.commit()
 
+            req_url = self.request.full_url()
+            location = f"{paste.paste_id}#{url_escape(paste.description)}"
```

Comment:
Also asking for better way.

Reply:
This is the correct way to do this. You could roll this into line 222 and 223 if you want so you don't have extra assignments but this is perfectly readable.